README.md

```diff
@@ -14,6 +14,7 @@ The committee consists of 6 members:
 * Emily Pillmore @emilypi (term ending January 2023)
 * Melanie Brown @cigsender (term ending January 2025)
 * Andrew Lelechenko @Bodigrim (chair, term ending January 2025)
+* Tom Ellis @tomjaguarpaw (term ending February 2025)
 
 The committee makes decisions by simple majority voting.
 
```

Comment:
Gah.  I want to add a comment on line 23, but GItHub won't let me.  It says "Members of the public are suggested to contact CLC primarily via this issue tracker".  But what issue tracker?  This is the first mention of such a thing.

If you mean the base-package-API-propsals issue tracker, that wouldn't be appropriate for stuff about PVP or core-library maintainers.  Or are there two issue trackers?